src/time_travel/time_machine_clock.py

```diff
@@ -1,11 +1,18 @@
 """A single clock for all patchers."""
 
 
+MIN_START_TIME = 100000.0  # Windows support unix times starting from 86,400
```

Comment:
This should be exported as library API by the main __init__.py file.

Reply:
Done